UACloudLibraryServer/Controllers/InfoModelController.cs

```diff
@@ -123,6 +123,35 @@ public IActionResult GetAllNamesandIdentifiersAsync()
             return new ObjectResult(result) { StatusCode = (int)HttpStatusCode.OK };
         }
 
+        [HttpGet]
+        [Route("/infomodel/downloadNodeSetXml/{identifier}")]
+        [SwaggerResponse(statusCode: 200, type: typeof(UANameSpace), description: "The OPC UA Information model and its metadata.")]
```

Comment:
How is this different from /infomodel/download/

Reply:
I have removed that duplicate function and added a parameter to only retrieve the NodeSetXML

Reply:
@barnstee Can you have a look at the things I have changed? Thanks.